start.rb

```diff
@@ -7,7 +7,7 @@
 	config.access_token_secret = ENV["ACCESS_TOKEN_SECRET"]
 end
 
-#client = LocalClient.new
+# client = LocalClient.new
```

Comment:
Instead of always checking to see if this is set, we should have a check to see if we're in production or development. If in production then use twitter, else if in development use local client.